src/common/options.cc

```diff
@@ -4856,11 +4856,11 @@ std::vector<Option> get_rgw_options() {
     .set_default(0)
     .set_description(""),
 
-    Option("rgw_period_push_interval", Option::TYPE_FLOAT, Option::LEVEL_ADVANCED)
+    Option("rgw_period_push_interval", Option::TYPE_INT, Option::LEVEL_ADVANCED)
     .set_default(2)
```

Comment:
I don't know if we can do this in Jewel, if anyone is using a time of .25 seconds or 1.5 seconds or whatever it'll break their configuration. It may be better to leave it as a double and use `make_timespan`.